### PR TITLE
Removed Old and Busted Center Tags

### PR DESCRIPTION
**Pull request by stanidesis**
Closes [#473](https://github.com/Bloc/workshop/issues/473)

### CODE REVIEW

File: resources/javascript/functions.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ add(3, 5); // returns 8
 
 This is known as a __function expression__. Assigning a value to a variable is an expression, thus assigning a function to a variable must end in a semicolon.
 
-<center>![function expression](https://bloc-global-assets.s3.amazonaws.com/images-frontend/screenshots/13-functions/function_expression.png)</center>
+![function expression](https://bloc-global-assets.s3.amazonaws.com/images-frontend/screenshots/13-functions/function_expression.png)
```
Comment (bmneely) [inline resources/javascript/functions.md]: This one has no meta data and won't blync.

File: resources/javascript/objects.md [hunk 2]
```diff
@@ -197,7 +197,7 @@ orange.color; // returns "red"
 
 By assigning `orange.color` a value of `"red"`, we overwrite the previous value of `"orange"`.
 
-<center>![update object property value](https://bloc-global-assets.s3.amazonaws.com/images-frontend/screenshots/15-objects/objects_prop_update.gif)</center>
+![update object property value](https://bloc-global-assets.s3.amazonaws.com/images-frontend/screenshots/15-objects/objects_prop_update.gif)
```
Comment (bmneely) [inline resources/javascript/objects.md]: Also no meta for this one.

File: resources/javascript/operators.md [hunk 3]
```diff
@@ -121,7 +121,7 @@ __String concatenation__ is the act of combining two or more strings. There are
 
 The following video demonstrates the use of strings and the usefulness of concatenation:
 
-<center><iframe src="//fast.wistia.net/embed/iframe/yhe01w2geo" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" allowfullscreen mozallowfullscreen webkitallowfullscreen oallowfullscreen msallowfullscreen width="100%" height="300"></iframe></center><script src="//fast.wistia.net/assets/external/E-v1.js" async></script>
+<iframe src="//fast.wistia.net/embed/iframe/yhe01w2geo" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" allowfullscreen mozallowfullscreen webkitallowfullscreen oallowfullscreen msallowfullscreen width="100%" height="300"></iframe><script src="//fast.wistia.net/assets/external/E-v1.js" async></script>
```
Comment (bmneely) [inline resources/javascript/operators.md]: Or this one

**Comment by bmneely** [comment]
Looks good @stanidesis. I'm not sure all of them will deploy via blync.

**Comment by stanidesis** [comment]
@bmneely so those appear to be in curriculum (regular curriculum) and not within Workshop itself. Not sure what to do with them... ehhh @clln?

**Comment by stanidesis** [comment]
@bmneely we're going to go ahead and merge this for now and close the related issue, @clln will work with the meta-data when she gets back.

**Comment by robmadden** [comment]
# Blync Deploy!
## The following files were deployed
* [courses/javascript-primer/05-objects/07-change-a-value/body.md](https://bloc-workshop.herokuapp.com/api/v1/exercises/180)

## The following files were not deployed
* resources/javascript/closures.md - {"error":"Record not found","status":404}
* resources/javascript/conditionals.md - {"error":"Record not found","status":404}
* resources/javascript/functions.md - resources/javascript/functions.md file contains no meta data
* resources/javascript/loops.md - {"error":"Record not found","status":404}
* resources/javascript/objects.md - resources/javascript/objects.md file contains no meta data
* resources/javascript/operators.md - resources/javascript/operators.md file contains no meta data
* resources/javascript/variables.md - resources/javascript/variables.md file contains no meta data
* resources/ruby/command-line.md - resources/ruby/command-line.md file contains no meta data
* resources/ruby/git.md - resources/ruby/git.md file contains no meta data
* resources/ruby/github.md - resources/ruby/github.md file contains no meta data

**Pull request merged**